scripts/Muon/GUI/Common/fitting_widgets/tf_asymmetry_fitting/tf_asymmetry_fitting_model.py

```diff
@@ -716,6 +716,16 @@ def get_all_fit_functions(self) -> list:
         else:
             return super().get_all_fit_functions()
 
+    def get_all_fit_functions_for(self, display_type: str) -> list:
+        """Returns all the fit functions for datasets with a name containing a string."""
```

Comment:
could we use the fact that the filter only exists for transform data to simplify this? As we know the TF asymmetry data will never need a filter (always time domain)

Reply:
Yes, I can remove this

Reply:
Actually I just realised I can't remove this, but I can modify it so that the tf asymmetry fit functions do not get filtered

Reply:
I think we can just move one of the if/else statements in the function.

Reply:
I have removed the inner if statement as I can just return `self.get_all_fit_functions()` for TF asymmetry mode